bundles/org.openhab.binding.solarforecast/README.md

```diff
@@ -0,0 +1,340 @@
+Forecasts are delivered up to 3 days for paid personal plans.
+
+
+| Channel                 | Type          | Unit | Description                                     | Advanced |
+|-------------------------|---------------|------|-------------------------------------------------|----------|
+| power-estimate          | Number:Power  | W    | Power forecast for next hours/days              | no       |
+| energy-estimate         | Number:Energy | kWh  | Energy forecast for next hours/days             | no       |
+| power-actual            | Number:Power  | W    | Power prediction for this moment                | no       |
+| energy-actual           | Number:Energy | kWh  | Today's forecast till now                       | no       |
+| energy-remain           | Number:Energy | kWh  | Today's remaining forecast till sunset          | no       |
+| energy-today            | Number:Energy | kWh  | Today's forecast in total                       | no       |
+| json                    | String        | -    | Plain JSON response without conversions         | yes      |
+
+
+## Thing Actions
+
+All things `sc-site`, `sc-plane`, `fs-site` and `fs-plane` are providing the same Actions.
+Channels are providing actual forecast data and daily forecasts in future.
+Actions provides an interface to execute more sophisticated handling in rules.
+You can execute this for each `xx-plane` for specific plane values or `xx-site` to sum up all attached planes.
+
+See [Date Time](#date-time) section for more information.
+Double check your time zone in *openHAB - Settings - Regional Settings* which is crucial for calculation.
+
+### `getForecastBegin`
+
+Returns `Instant` of the earliest possible forecast data available.
+It's located in the past, e.g. Solcast provides data from the last 7 days.
+`Instant.MAX` is returned in case of no forecast data is available.
+
+### `getForecastEnd`
+
+Returns `Instant` of the latest possible forecast data available.
+`Instant.MIN` is returned in case of no forecast data is available.
+
+### `getPower`
+
+| Parameter | Type          | Description                                                                                |
+|-----------|---------------|--------------------------------------------------------------------------------------------|
+| timestamp | Instant       | Timestamp of power query                                                                   |
+| mode      | String        | Choose `average`, `optimistic` or `pessimistic` to select forecast scenario. Only Solcast. |
+
+Returns `QuantityType<Power>` at the given `Instant` timestamp.
+Respect `getForecastBegin` and `getForecastEnd` to get a valid value.
+
+Check log or catch exceptions for error handling
+
+- `IllegalArgumentException` thrown in case of problems with call arguments
+- `SolarForecastException` thrown in case of problems with timestamp and available forecast data
```

Comment:
Just to be sure, since I haven't tried throwing exceptions from actions: Are they only caught and logged by the rule engine, or can they be caught by rules? Which automations did you test with?